src/components/item/item-reorder-gesture.ts

```diff
@@ -134,7 +135,7 @@ export class ItemReorderGesture {
   }
 
   private itemForCoord(coord: PointerCoordinates): HTMLElement {
-    const sideOffset = this.plt.isRTL ? 100 : -100;
+    const sideOffset = isRightSide(this.reorderList.side, this.plt.isRTL) ? -100 : 100;
```

Comment:
could we cache some "_reorderSide" as a boolean (probably renamed to (_isRightSide) that is calculated in ngOnInit()?